agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4416,27 +4416,60 @@ bool slave_thread::autoconfig_wsc_parse_m2_encrypted_settings(
     return true;
 }
 
+/**
+ * @brief Parse AP-Autoconfiguration WSC which should include one AP Radio Identifier
+ *        TLV and one or more WSC TLV containing M2
+ * @param sd socket descriptor
```

Comment:
In general, there should be an empty line after the brief to signal the end of the paragraph. In this case, the param handles that, but it's cleaner to always have an empty line.